hello_world/pip-lister.py

```diff
@@ -0,0 +1,80 @@
+import pkg_resources
+from rich import box
+from rich import print
+from rich.console import Console
+from rich.table import Table
+
+
+def pip_freeze(table: Table, highlight: str = "rich") -> None:
+    """
+    docstring.
+
+    12.
+    """
+    for pkg in pkg_resources.working_set:
+        if pkg.key == highlight:
+            table.add_row(pkg.key, pkg.version, style="bold red on yellow")
+        else:
+            table.add_row(pkg.key, pkg.version)
+
+
+def get_table() -> None:
+    """
+    docstring.
+
+    12.
+    """
+    table = Table(title="Python Packages", box=box.DOUBLE_EDGE)
+    table.add_column("Name", justify="right", style="cyan", no_wrap=False)
+    table.add_column("Version", style="magenta")
+    return table
```

Comment:
`-> None` is the return type annotation i.e. what is the `type()` of the object returned by the function, here you return `table` which is of type `Table`.
```suggestion
def get_table() -> Table:
    """
    docstring.

    12.
    """
    table = Table(title="Python Packages", box=box.DOUBLE_EDGE)
    table.add_column("Name", justify="right", style="cyan", no_wrap=False)
    table.add_column("Version", style="magenta")
    return table
```

Reply:
got it thanks.